src/microsoft-trydotnet-editor/tests/apiServiceSimulator.ts

```diff
@@ -48,7 +48,6 @@ function patchEvents(events: polyglotNotebooks.KernelEventEnvelopeModel[], comma
         let event = events[i];
         var sourceCommand = commands.find(command => command.commandType === event.command!.commandType);
         let patchedEvent = { ...event };
-        patchedEvent.command!.id = sourceCommand!.id;
```

Comment:
Could you share some context on why this was removed please?

Reply:
this concept was removed.